### ISSUE TITLE
Build timelines and paths for publication of all white papers

### ISSUE THREAD

**Issue by RaoOfPhysics** [alan-turing-institute/asg-community#32]
- Work with @malvikasharan
- Think back from the deadlines and plan milestones

**Comment by RaoOfPhysics** [comment]
_Contributions from other authors have been made primarily to the **ASG Contributions** section, and these notes are now being consolidated into narrative. We need more help/contributions for the **Introduction** and **Challenges** section, in stringing together a narrative. Zeynep Engin has provided some comments in the document._

# Contributions made per section

## Introduction

- [Work needed]

## Challenges

- Evangeline Corcoran
- [Work needed]

## ASG challenges

### Scivision

- Evangeline Corcoran
- Beatriz Costa Gomes
- Sebastian Ahnert
- Alan Lowe

### IceNet

- Tom Andersson
- James Byrne

### DyME-CHH

- Ruth Bowyer

### Impact of Climate Change on UK Agriculture and Food Security

- Evangeline Corcoran
- Sebastian Ahnert

## Recommendations

- [Work needed]

**Comment by malvikasharan** [comment]
**Timeline update discussion: 23 June 2022**

- need to create the first draft of the E&S white paper based on the current info and start getting the reviews
  - Lots of notes are already in there as noted above
- What other work will need to be deprioritised or have less engagement from your side?
  - AI-UK 2023
  - Will not be involved in the organising of the nature workshop
  - The July event planning work has already been taken off by ZI
  - MS will document process to be added on this GitHub repo
- What support do you need?
  -  ZI has been a huge help (follow up and admin support), will have a planning sprint.
  - Initially the plan was that the core team will contribute, that has been challenging - need stronger steers from BM in places where engaging with senior/busy people is hard
- Who else can help you with it?
  -  Let's get RAMs, TPS PM and CMs involved in reviewing  - let them know that next 10 days are going to be tough.

**Comment by ZaynabIsmail** [comment]
From my perspective I'm still on the edge of all the discussions related to the whitepapers.  I am keen to help, I'll let @RaoOfPhysics let me know how I can support the work and when to draw me in.  

In terms of actual project management of the E&S paper as it is in a phase of 'write up' by Achintya and Zeynep there is not a lot for me to manage or support at the moment.  Perhaps I can help with the reviewing mentioned above?Please let me know if i am incorrect.  Perhaps I can help with the reviewing mentioned above?

For the other papers I can definitely help in the initial stages of liaising between the contributors and their line management.

**Comment by ZaynabIsmail** [comment]
@malvikasharan, @RaoOfPhysics and @hsweeney164 the detailed timeline for the E&S white paper has been created and open for review and edits.

**Please let me know:**
1) If the dates are acceptable?
2) If any further actions need to be added?
3) If the owners are correct?
4) How the comms team may need to be included and in what way.

**Note:** Based on how much time this single document will take to produce I am a little concern regarding the other two.  Some thought and discussion on how the December deadline can be met may be needed.

**Draft E&S White paper timeline**[ link](https://thealanturininstitute.sharepoint.com/:x:/s/rid/EUBoqKT2dpJGhcvUTD2QoBUB6cqrXJtzm7QBBugxipcokQ?e=rNcybz).

**Comment by ZaynabIsmail** [comment]
@RaoOfPhysics I will need confirmation from you as to the internal and external reviewers.  Please review the current list with Ben this week and let me know who I should be reaching out to and securing.

- [ ] @RaoOfPhysics to confirm the internal and external reviewers for the E&S white paper
- [ ] @RaoOfPhysics to confirm suitable time to initiate the cycle for the next white paper

**Comment by RaoOfPhysics** [comment]
Bouncing the list we settled on with Ben and will get back to you on that.

Can you please remind me to get back to you about the timeline for the next two white papers?

**Comment by ZaynabIsmail** [comment]
Yes can I book in another 3 hr session, to cover this anything else we didn't discuss last time?  We may not need it all.

**Comment by malvikasharan** [comment]
Thanks for this @ZaynabIsmail and @RaoOfPhysics. I will look at this later today.
